app/src/main/java/com/eventyay/organizer/core/event/dashboard/EventDashboardFragment.java

```diff
@@ -208,7 +208,11 @@ public void showOrderStatistics(OrderStatistics orderStatistics) {
 
     @Override
     public void showError(String error) {
-        ViewUtils.showSnackbar(binding.getRoot(), error);
+        if (error.contains("starts-at")) {
```

Comment:
You need to properly parse the error

Reply:
Not understood. Pls, elaborate.

Reply:
starts-at can be in any error unrelated to past events

Reply:
Do I need to match the exact error coming from the backend?

Reply:
OK